cg-spec/editors_draft.html

```diff
@@ -2229,38 +2238,33 @@ <h2>Merging RDF-star graphs</h2>
 
-        <p>Furthermore, it is expected that some implementations will not rely on the mapping, but represent and work directly with the <a href="#concepts">abstract syntax</a> of RDF-star. For these implementations, having to handle both the native and the mapped representation of <a>embedded triples</a> would be even more challenging.</p>
+        <p>Conversely, nothing in the definition of |unstar| prevents it from generating the same blank node |b| for an <a>embedded triple</a> of |G|, and for a <em>different</em> <a>embedded triple</a> of |H| (provided that |b| is not present in either |G| nor |H|). In that case, not only would |unstar|(|G|) ∪ |unstar|(|H|) be different from |unstar|(|G| ∪ |H|), but it would lose some information by mixing the descriptions of the two <a>embedded triples</a>.</p>
 
-        <pre id="unrestricted-use-of-ST-and-SS"
-        data-transform="updateExample"
-        data-content-type="text/x-turtle-star"
-        class="nohighlight example"
-        >
-          <!--
-          :alice :says << :bob :age 42 >>.
-          << :bob :age 42 >> :ST :charlie;
-                             :SS "<http://example.org/charlie>".
+      </section>
 
-          # assuming that :ST and :SS stand for the corresponding special IRIs,
-          # the graph above entails the graph below
+      <section>
+        <h2>Other uses of the |unstar| mapping</h2>
 
-          :alice :says << :charlie :age 42 >>.
-          -->
-        </pre>
+        <p>Since the |unstar| mapping defined <a href="#mapping">above</a> transforms any <a>RDF-star graph</a> into a standard ("non-star") <a>RDF graph</a>, it might be tempting to use this mapping for conveying <a>RDF-star graphs</a> to legacy RDF systems. This however has a number of caveats.</p>
+
+        <p>First, the |unstar| mapping is altering the semantics of the graph: in general, a graph |G| is <em>not</em> <a>equivalent</a> to its "unstarred" version |unstar|(|G|), even if |G| is a standard <a>RDF graph</a> containing no <a>embedded triple</a>. (More precisely, they are equivalent only if |G| contains no <a>embedded triple</a> and no IRI from the `unstar:` namespace.)</p>
+
+        <p>Second, when a legacy RDF system communicates an <a>RDF graph</a> to an RDF-star-aware system, there is no mean for the latter to determine with certainty whether the graph is the result of the |unstar| mapping, and thus needs to be transformed back. (The presence of `unstar:` IRIs in the graph can be used as a hint, but such IRIs can also be used independantly of the mapping.) Additional metadata should therefore be attached to unstarred graphs, so that RDF-star systems know whether they need to apply or not the inverse transformation of |unstar|.</p>
```

Comment:
```suggestion
        <p>Second, when a legacy RDF system communicates an <a>RDF graph</a> to an RDF-star-aware system, there is no means by which the latter can determine with certainty whether the graph is the result of an |unstar| mapping, and thus needs to be transformed back. (The presence of `unstar:` IRIs in the graph can be used as a hint, but such IRIs can also be used independently of the mapping.) Additional metadata should therefore be attached to unstarred graphs, so that RDF-star systems know whether  or not they need to apply the inverse transformation of |unstar|.</p>
```